code/modules/atmospherics/machinery/other/miner.dm

```diff
@@ -0,0 +1,152 @@
+/obj/machinery/atmospherics/miner
+	name = "gas miner"
+	desc = "Gasses mined from the gas giant below (above?) flow out through this massive vent."
```

Comment:
But we have Lavaland planet now. And soon-possibly-maybe the station will be ON the planet!